### ISSUE TITLE
fail to build on windows

### ISSUE THREAD

**Issue by YongHee-Kim** [zyedidia/Chipmunk.jl#3]
my environmental is windows7 with cmake and MingGW installed for GNU support. 
`+visual studio community 2015 as compiler

build.jl fails at line 25
`run('make')`

I assume that this is due to cmke configuration error, which suppose to generate makefile but didn't 

here is full log message 

> Checking dependencies...
> Good...
> Cloning Chipmunk source...
> Cloning into 'Chipmunk2D'...
> Note: checking out 'Chipmunk-7.0.0'.
> 
> You are in 'detached HEAD' state. You can look around, make experimental
> changes and commit them, and you can discard any commits you make in this
> state without impacting any branches by performing another checkout.
> 
> If you want to create a new branch to retain commits you create, you may
> do so (now or later) by using -b with the checkout command again. Example:
> 
>   git checkout -b <new-branch-name>
> 
> HEAD is now at 079a859... Finish the build scripts.
> HEAD is now at 079a859... Finish the build scripts.
> -- Building for: Visual Studio 14 2015
> -- The C compiler identification is MSVC 19.0.23918.0
> -- The CXX compiler identification is MSVC 19.0.23918.0
> -- Check for working C compiler using: Visual Studio 14 2015
> -- Check for working C compiler using: Visual Studio 14 2015 -- works
> -- Detecting C compiler ABI info
> -- Detecting C compiler ABI info - done
> -- Check for working CXX compiler using: Visual Studio 14 2015
> -- Check for working CXX compiler using: Visual Studio 14 2015 -- works
> -- Detecting CXX compiler ABI info
> -- Detecting CXX compiler ABI info - done
> -- Detecting CXX compile features
> -- Detecting CXX compile features - done
> Configuring Chipmunk2D version 7.0.0
> -- Configuring done
> -- Generating done
> -- Build files have been written to: C:/Users/Yonghee/.julia/v0.4/Chipmunk/deps/Chipmunk2D
> make: **\* No targets specified and no makefile found.  Stop.

**Comment by zyedidia** [comment]
Yeah this seems like a problem with cmake. First make sure this directory exists `C:/Users/Yonghee/.julia/v0.4/Chipmunk/deps/Chipmunk2D`.

If so, is there a Makefile in that directory?

**Comment by YongHee-Kim** [comment]
Yes `C:/Users/Yonghee/.julia/v0.4/Chipmunk/deps/Chipmunk2D`exist
and makefile dosen't exist

It turns out using visual studio 15 as generator for cmake dosen't generate makefile. I should change generator.

**Comment by YongHee-Kim** [comment]
It turns out using visual studio 15 as generator for cmake doesn't generate makefile. 

So I've changed generator with `cmake -G "MSYS Makefiles"` command and generated makefile. 

but `make` still fails with following error

> [35m[1mScanning dependencies of target chipmunk[0m
> [  1%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/chipmunk.c.obj[0m
> [  3%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpArbiter.c.obj[0m
> [  4%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpArray.c.obj[0m
> [  6%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpBBTree.c.obj[0m
> [  7%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpBody.c.obj[0m
> [  9%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpCollision.c.obj[0m
> [ 10%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpConstraint.c.obj[0m
> [ 12%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpDampedRotarySpring.c.obj[0m
> [ 14%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpDampedSpring.c.obj[0m
> [ 15%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpGearJoint.c.obj[0m
> [ 17%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpGrooveJoint.c.obj[0m
> [ 18%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpHashSet.c.obj[0m
> [ 20%] [32mBuilding C object src/CMakeFiles/chipmunk.dir/cpHastySpace.c.obj[0m
> c:/Users/Yonghee/.julia/v0.4/Chipmunk/deps/Chipmunk2D/src/cpHastySpace.c:7:21: fatal error: pthread.h: No such file or directory
>  #include <pthread.h>
>                      ^
> compilation terminated.
> make[2]: **\* [src/CMakeFiles/chipmunk.dir/cpHastySpace.c.obj] Error 1
> make[1]: **\* [src/CMakeFiles/chipmunk.dir/all] Error 2
> make: **\* [all] Error 2
